app/assets/src/components/views/TaxonTreeVis.jsx

```diff
@@ -92,6 +98,18 @@ class TaxonTreeVis extends React.Component {
     this.setState({ nodeHover: node });
   }
 
+  onNodeLabelClick(node) {
```

Comment:
I have been trying to enforce a `handle...` prefix - like `handleNodeLabelClick` - for event handlers.

Same for function below.